sodapy/__init__.py

```diff
@@ -349,6 +349,12 @@ def close(self):
         '''
         self.session.close()
 
+    def set_cookie(self, cookie):
+        self.session.headers.update({"Cookie": cookie})
+
+    def set_ssl_verification(self, bool):
+        self.ssl_verify = bool
```

Comment:
You probably shouldn't use `bool` as that's the built-in function for converting things into booleans.

That and `bool` isn't a very descriptive name, I'd probably go with `enabled`.